release-notes.txt

```diff
@@ -1,6 +1,9 @@
 ---- Friday 3rd April, 2020 ----
 * Disclaimer screen on first run of the app, updated, fixes and supports versioning of the disclaimer
 
+---- Thursday 2nd April, 2020 ----
```

Comment:
Might want to move this to Saturday, as thats when the actual changes will happen in the app on testflight, etc 🙂